cirq/google/calibration/phased_fsim.py

```diff
@@ -261,6 +277,11 @@ class FloquetPhasedFSimCalibrationRequest(PhasedFSimCalibrationRequest):
 
     options: FloquetPhasedFSimCalibrationOptions
 
+    @classmethod
+    def from_moment(cls, moment: Moment, options: FloquetPhasedFSimCalibrationOptions):
```

Comment:
docstring, especially since there are restrictions on what `moment` can be, namely all the same two-qubit gate

Reply:
+1, especially for explanation on what moments are allowed

Reply:
Done.